src/components/partner/dto/update-partner.dto.ts

```diff
@@ -0,0 +1,28 @@
+import { Field, InputType, ObjectType } from '@nestjs/graphql';
+import { Type } from 'class-transformer';
+import { ValidateNested } from 'class-validator';
+import { IdField } from '../../../common';
+import { Partner } from './partner';
+
+@InputType()
+export abstract class UpdatePartner {
+  @IdField()
+  readonly id: string;
+
+  @IdField({ nullable: true })
+  readonly organizationId?: string;
```

Comment:
Mmm I don't think we want to be able to change organization on update. It's a fundament part of the relationship. Just like we don't allow changing the language or project on a language engagement.